crypto/rand/rand_lcl.h

```diff
@@ -44,7 +52,8 @@
  */
 # define DRBG_MAX_LENGTH                         INT32_MAX
 
-
+/* The default nonce */
+# define DRBG_DEFAULT_NONCE                      "OpenSSL NIST SP 800-90A DRBG"
```

Comment:
This is not used as a nonce, it's used as a personalisation string.

Reply:
That's a pretty bad oops.  I'll fix.